Gemfile

```diff
@@ -0,0 +1,9 @@
+
+source 'https://rubygems.org'
+
```

Comment:
Yeah, I still haven't found a satisfying way to deal with this in generated files.